test/functional/p2p_add_connections.py

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env python3
+# Copyright (c) 2020 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+"""Test add_outbound_p2p_connection test framework functionality"""
+
+from test_framework.test_framework import BitcoinTestFramework
+from test_framework.p2p import P2PInterface
+from test_framework.util import assert_equal
+
+
+class P2PAddConnections(BitcoinTestFramework):
+    def set_test_params(self):
+        self.setup_clean_chain = False
+        self.num_nodes = 2
+
+    def skip_test_if_missing_module(self):
+        self.skip_if_no_wallet()
+
+    def run_test(self):
+        self.disconnect_nodes(0, 1)
+
+        self.log.info("Add 8 outbounds to node 0")
+        for i in range(8):
+            self.log.info("outbound: {}".format(i))
+            self.nodes[0].add_outbound_p2p_connection(P2PInterface())
```

Comment:
50720f411e05 Suggest also testing passing "outbound" in one of these calls (and one with e.g. "  OUTbound " if you make it case-insensitive and trimmed)

Reply:
added outbound to one and BlockRelay to another